src/_includes/partials/meta.njk

```diff
@@ -0,0 +1,64 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+
+{% if title and number %}
+  {% set pageTitle %}{{ number }} {{ title }} - {{ site.title }}{% endset %}
+{% elif title %}
+  {% set pageTitle %}{{ title }} - {{ site.title }}{% endset %}
+{% else %}
+  {% set pageTitle %}{{ site.title }}{% endset %}
+{% endif %}
+
+<title>{{ pageTitle }}</title>
+
+<link rel="icon" href="{{ site.baseURL }}/favicon.ico">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.3.2/styles/a11y-light.min.css">
+
+{% if pageType === "default" %}
```

Comment:
ローカル起動している際ってCSS参照できてますよね？
自分の環境だと

```
Refused to apply style from 'http://localhost:3000/a11y-guidelines/css/default.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

という感じで怒られてしまうのですがビルドはちゃんとされていそうでして。

Reply:
できてますYO。ちょっとみてみますー